src/Maths/Silk.NET.Intrinsics/Avx/AvxRegister.Float.cs

```diff
@@ -0,0 +1,330 @@
+        }
+
+        public WorkUnit<float> MultiplySubtractFast(WorkUnit<float> x, WorkUnit<float> y, WorkUnit<float> z)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> NegateMultiplySubtractFused(WorkUnit<float> x, WorkUnit<float> y, WorkUnit<float> z)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> NegateMultiplySubtractFast(WorkUnit<float> x, WorkUnit<float> y, WorkUnit<float> z)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Divide(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Sqrt(WorkUnit<float> vector)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Max(WorkUnit<float> vector)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> Min(WorkUnit<float> vector)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> GreaterThan(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> LessThan(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> GreaterThanOrEqualTo(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public WorkUnit<float> LessThanOrEqualTo(WorkUnit<float> left, WorkUnit<float> right)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public unsafe void StoreScalar(WorkUnit<float> vector, float* ptr)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public unsafe void StoreVector2(WorkUnit<float> vector, float* ptr)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public unsafe void StoreVector3(WorkUnit<float> vector, float* ptr)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public unsafe void StoreVector4(WorkUnit<float> vector, float* ptr)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public unsafe WorkUnit<float> ToScalar(float* ptr)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public unsafe WorkUnit<float> ToVector2(float* ptr)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public unsafe WorkUnit<float> ToVector3(float* ptr)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public unsafe WorkUnit<float> ToVector4(float* ptr)
```

Comment:
Is this supposed to be a `Load` counterpart to `Store`?

Reply:
Yeah, probably poorly named.